scapy/arch/common.py

```diff
@@ -10,15 +10,29 @@
 # Important Note: This file is not needed on Windows, and mustn't be loaded
 
 import socket
+import subprocess
 from fcntl import ioctl
 import os
 import struct
 import ctypes
 from ctypes import POINTER, Structure
 from ctypes import c_uint, c_uint32, c_ushort, c_ubyte
 from scapy.config import conf
+from scapy.data import MTU
+from scapy.error import log_loading, Scapy_Exception
 import scapy.modules.six as six
 
+# BOOT
+
+with os.popen("%s -V 2> /dev/null" % conf.prog.tcpdump) as fdesc:
```

Comment:
Would you mind replacing that with a `subprocess.Popen()` call?

Reply:
Done !